gnucash/report/report-system/html-utilities.scm

```diff
@@ -818,6 +818,65 @@
        "")
       (_ "Edit report options")))))
 
+(define* (gnc:render-options-changed options #:optional plaintext?)
```

Comment:
Good choice for the new location.
I would suggest updating the name to start with "gnc:html" to be more consistent with the surrounding functions. That makes it slightly easier to trace back the code.